tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/im2latex_evaluator.py

```diff
@@ -0,0 +1,302 @@
+    launcher_model_mapping = {
+        'dlsdk': RecognizerDLSDKModel
+    }
+    framework = launcher.config['framework']
+    model_class = launcher_model_mapping.get(framework)
+    if not model_class:
+        raise ValueError('model for framework {} is not supported'.format(framework))
+    return model_class(model_config, launcher, suffix)
+
+
+class SequentialModel:
+    def __init__(self, network_info, launcher, models_args, meta, is_blob=None):
+        recognizer_encoder = network_info.get('recognizer_encoder', {})
+        recognizer_decoder = network_info.get('recognizer_decoder', {})
+        if 'model' not in recognizer_encoder:
+            recognizer_encoder['model'] = models_args[0]
+            recognizer_encoder['_model_is_blob'] = is_blob
+        if 'model' not in recognizer_decoder:
+            recognizer_decoder['model'] = models_args[len(models_args) == 2]
+            recognizer_decoder['_model_is_blob'] = is_blob
+        network_info.update({
+            'recognizer_encoder': recognizer_encoder,
+            'recognizer_decoder': recognizer_decoder
+        })
+        if not contains_all(network_info, ['recognizer_encoder', 'recognizer_decoder']):
+            raise ConfigError('network_info should contain encoder and decoder fields')
+        self.vocab = meta['vocab']
+        self.recognizer_encoder = create_recognizer(network_info['recognizer_encoder'], launcher, 'encoder')
+        self.recognizer_decoder = create_recognizer(network_info['recognizer_decoder'], launcher, 'decoder')
+        self.sos_index = 0
+        self.eos_index = 2
+        self.max_seq_len = int(network_info['max_seq_len'])
+
+    def get_phrase(self, indices):
+        res = ''
+        for idx in indices:
+            if idx != self.eos_index:
+                res += ' ' + str(self.vocab.get(idx, '?'))
+            else:
+                return res.strip()
+        return res.strip()
+
+    def predict(self, idenitifiers, input_data):
+        assert len(idenitifiers) == 1
+        input_data = np.array(input_data)
+        input_data = np.transpose(input_data, (0, 3, 1, 2))
+        enc_res = self.recognizer_encoder.predict(
+            inputs={'imgs': input_data})
+        row_enc_out = enc_res['row_enc_out']
+        dec_states_h = enc_res['hidden']
+        dec_states_c = enc_res['context']
+        O_t = enc_res['init_0']
+
+        tgt = np.array([[self.sos_index]])
+        logits = []
+        for _ in range(self.max_seq_len):
+
+            dec_res = self.recognizer_decoder.predict(inputs={'row_enc_out': row_enc_out,
+                                                              'dec_st_c': dec_states_c, 'dec_st_h': dec_states_h,
+                                                              'O_t_minus_1': O_t, 'tgt': tgt
+                                                              })
+
+            dec_states_h = dec_res['dec_st_h_t']
+            dec_states_c = dec_res['dec_st_c_t']
+            O_t = dec_res['O_t']
```

Comment:
Decoder model has no 'O_t' output
```suggestion
            O_t = dec_res['output']
```